packages/playground/windows/playground/HostingPane.xaml.cpp

```diff
@@ -318,6 +319,22 @@ void HostingPane::OnSelectionChanged_JavaScriptFilename(
   UpdateUI();
 }
 
+void HostingPane::OnTextChanging_ValidPort(TextBox ^ sender, TextBoxTextChangingEventArgs ^ args) {
+  std::wstring buffer = L"0";
+  for (auto wchar = sender->Text->Begin(); wchar != sender->Text->End(); ++wchar) {
+    if (iswdigit(*wchar)) {
```

Comment:
>iswdigit(*wchar) [](start = 8, length = 16)

The code could be simplified with use of the standard strtol or wcstol for the wide chars.
These functions report the last character read and by comparing the end_ptr with the end() of the range we can see if the conversion was successful.